resources/views/layouts/menu.blade.php

```diff
@@ -364,13 +364,14 @@
                         <li><a href="{{ url('authlog') }}"><i class="fa fa-shield fa-fw fa-lg" aria-hidden="true"></i> Auth History</a></li>
                         <li role="presentation" class="divider"></li>
                         <li class="dropdown-submenu">
-                            <a href="#"><i class="fa fa-th-large fa-fw fa-lg" aria-hidden="true"></i> Pollers</a>
```

Comment:
The one thing with this change is it makes the sub-menu unusable on a mobile. Clicking the Pollers drop down now just takes you to the pollers page so the sub items in the menu are now only accessible by going to the main pollers page and clicking through the tabs from there.

I'll merge anyway but personally, I think it would be good to restore that.